discovery/src/test/java/com/ibm/watson/developer_cloud/discovery/v1/DiscoveryServiceTest.java

```diff
@@ -483,124 +421,36 @@ public void createDocumentFromInputStreamWithMediaTypeIsSuccessful() throws Inte
   }
 
   @Test
-  public void createDocumentFromInputStreamWithFileNameAndMediaTypeIsSuccessful() throws InterruptedException {
+  public void addDocumentFromInputStreamWithFileNameAndMediaTypeIsSuccessful() throws InterruptedException {
     server.enqueue(jsonResponse(createDocResp));
     String fileName = "MyFileName";
     String myDocumentJson = "{\"field\":\"value\"}";
     JsonObject myMetadata = new JsonObject();
     myMetadata.add("foo", new JsonPrimitive("bar"));
     InputStream documentStream = new ByteArrayInputStream(myDocumentJson.getBytes());
 
-    CreateDocumentRequest.Builder builder = new CreateDocumentRequest.Builder(environmentId, collectionId);
-    builder.file(documentStream, fileName, HttpMediaType.APPLICATION_JSON);
-    builder.metadata(myMetadata);
-    CreateDocumentResponse response = discoveryService.createDocument(builder.build()).execute();
+    DocumentAccepted response = discoveryService.addDocument(environmentId, collectionId, null, documentStream,
+        HttpMediaType.APPLICATION_JSON, myMetadata.toString(), null).execute();
     RecordedRequest request = server.takeRequest();
 
     assertEquals(DOCS1_PATH, request.getPath());
     assertEquals(POST, request.getMethod());
     assertEquals(createDocResp, response);
   }
 
-  @Test
```

Comment:
Those tests used a `File` parameter rather than `InputStream`.  The variant of the API that accepts `File` is marked "deprecated", so I didn't want to write code to support it only to drop it in the near future.  So instead I just dropped the tests that specify a file.